Guide/src/user_guide/openhcl/run/hyperv.md

```diff
@@ -31,7 +31,7 @@ Set-ItemProperty "HKLM:/Software/Microsoft/Windows NT/CurrentVersion/Virtualizat
 ```
 
 ### File access
-Ensure that your OpenHCL .bin is located somewhere that vmwp.exe in your Windows host has permissions to read it (that can be in windows\system32, or another directory with wide read access). 
+Ensure that your OpenHCL .bin is located somewhere that vmwp.exe in your Windows host has permissions to read it (that can be in windows\system32, or another directory with wide read access).
```

Comment:
We should take a note to make the recommendation here more crisp (e.g. to give permission to the vmsid or vm group)

Reply:
If you have some precise wording changes you'd like to see, I could probably drive-by improve things in this PR. Otherwise, we can split this out into a follow-up

Reply:
Talked offline - sounds like there's a bit more workshopping to do here. Lets address this in a subsequent content-focused PR.